tests/components/mqtt/test_state_vacuum.py

```diff
@@ -316,15 +331,14 @@
 async def test_custom_availability_payload(hass, mqtt_mock):
     """Test availability by custom payload with defined topic."""
     config = deepcopy(DEFAULT_CONFIG)
-    config.update(
-        {
-            "availability_topic": "availability-topic",
-            "payload_available": "good",
-            "payload_not_available": "nogood",
-        }
-    )
+    config.update({
+        "availability_topic": "availability-topic",
+        "payload_available": "good",
+        "payload_not_available": "nogood",
+    })
 
-    assert await async_setup_component(hass, vacuum.DOMAIN, {vacuum.DOMAIN: config})
+    assert await async_setup_component(hass, vacuum.DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)